app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -122,6 +126,14 @@ public static BrowserFragment create(String url) {
     @Override
     public void onPause() {
         super.onPause();
+        getContext().unregisterReceiver(DownloadReceiver.getDownloadReceiver());
```

Comment:
This will be broken when there are multiple BrowserFragment instances. Its life cycle should be handled within BrowserFragment only.